apps/web/src/app/[locale]/[username]/page.tsx

```diff
@@ -54,16 +55,32 @@ export async function generateMetadata({ params: { username: usernameFromQuery }
     },
   });
 
-  const bio = user ? user.bio : '';
+  const challenges = await prisma.challenge.findMany({
+    where: {
+      submission: {
+        some: {
+          userId: user?.id,
+          isSuccessful: true,
+        },
+      },
+    },
+    select: {
+      name: true,
+    },
+  });
+  
   const avatar = user && user.image ? user.image : '';
+  const date = user?.createdAt as Date
+  const dateSince = getRelativeTime(date)
 
   if (username)
     return buildMetaForUser({
       username,
       title: `${username}'s profile | TypeHero`,
       description: `View the profile of ${username} on TypeHero.`,
-      bio,
+      challenges: challenges.length,
```

Comment:
we can remove the challenges code now i believe

Reply:
oh missed that
```suggestion
```